draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -102,12 +102,11 @@ has to verify that all disclosed claim values were part of the original
 Issuer-signed JWT. The Verifier will not, however, learn any claim
 values not disclosed in the Disclosures.
 
-This document also specifies an optional mechanism for Key Binding,
-which is the concept of binding an SD-JWT to a Holder's public key
-and requiring that the Holder prove possession of the corresponding
-private key when presenting the SD-JWT.
-The strength of the binding is conditional upon the trust
-in the protection of the private key of the key pair an SD-JWT is bound to.
+For SD-JWTs where the issuer has specified an associated public key for the
+Holder (e.g., using the `cnf` claim [@?RFC7800]), this document also defines a
```

Comment:
```suggestion
For SD-JWTs where the issuer is binding as SD-JWT to a public key associated to the
Holder (e.g., using the `cnf` claim [@?RFC7800]), this document also defines a
```

Reply:
"is binding as SD-JWT to a" should be "is binding an SD-JWT to a"?